BC.HelperFunctions.ps1

```diff
@@ -120,6 +120,9 @@ function Get-ContainerHelperConfig {
             "MSAppsNuGetFeedUrl" = 'https://dynamicssmb2.pkgs.visualstudio.com/DynamicsBCPublicFeeds/_packaging/MSApps/nuget/v3/index.json'
             "TrustedNuGetFeeds" = @(
             )
+            "IsGitHubActions" = [Boolean]$env:GITHUB_ACTIONS
```

Comment:
In PowerShell - [boolean]"false" is also true - I would prefer 

"IsGitHubActions" = ($env:GITHUB_ACTIONS -eq "true")

(and the same with the other)

Reply:
fixed